api/task_delete_test.go

```diff
@@ -37,13 +37,13 @@ func TestTaskDelete_DeleteTaskByName(t *testing.T) {
 			http.StatusNotFound,
 		},
 		{
-			"task_is_running",
+			"task_errored",
```

Comment:
nit, I see that we're doing it here for consistency, but in general the name can include spaces. `gotest` handles them nicely when generating the test names the test cases within a unit test

Reply:
I started preferring `_` over spaces since I found it was easier to search for the test code when I had a failure since go test prints it as `_`. But I also know we don't do this everywhere, so maybe something to settle on as a team!